src/i18n/sv.json

```diff
@@ -71,5 +71,36 @@
     "panelTitle": "Vill du ladda ner dina uppgifter?",
     "panelText": "Du kan ladda ner all information vi har om dig från följande knapp.",
     "button": "Ladda ner informationen om mig"
+  },
+  "footer": {
+    "copyright": "Upphovsrätt {{year}}",
+    "reserveRights": "Alla rättigheter förbehållna",
+    "privacy": "Integritetspolicy",
+    "accessibility": "Tillgänglighetsdokument",
+    "terms": "Användarvillkor"
+  },
+  "tos": {
+    "title": "Helsinkiprofiilin käyttöä koskevat käyttöehdot",
```

Comment:
It's a bit strange that title2 has the text 1. XXX. Could you rename some of these titles and messages?
E.g. something like this:
```
"pageTitle": "Helsinkiprofiilin käyttöä koskevat käyttöehdot",
...
...
"termsTitle": "Helsinki profiili - avain kaikkiin kaupungin palveluihin",
"termsText": "Luomalla Helsinkiprofiilin voit...",
"terms1Title": "1. Soveltamisala",
"terms1Terms": "Helsinkiprofiili on Helsingin kaupungin..."
```

Reply:
Changed all the keys to match title numbers.